src/containers/mobile-screen-fallback.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { useWindowWidth } from '@react-hook/window-size'
+
+import indexStyles from '../css/index.css'
+import LazyRender from './lazy-render.jsx'
+
+function MobileScreenFallback(props) {
+  const width = useWindowWidth()
+  const isMobile = width < 600
+
+  return (
+    <React.Fragment>
```

Comment:
Can't you use `<>...</>` here?

Reply:
I can and I just did! I also replaced all other occurances of `React.Fragment` :)